slash/react/src/Form/core/Field.tsx

```diff
@@ -62,41 +127,60 @@ export const Field = ({
     actualRequired,
   );
 
+  const labelId = useId();
+
   if (!isVisible) {
     return null;
   }
 
+  const isGroup = roleContainer === "radiogroup" || roleContainer === "group";
+  const LabelElement = isGroup ? "div" : "label";
+
+  const modifiers = forceDisplayMessage
+    ? `${inputFieldClassModifier} ${FormClassManager.getModifier(messageType)}`
+    : inputFieldClassModifier;
   const fieldContainerClassName = getComponentClassName(
-    "af-form__text",
-    forceDisplayMessage
-      ? `${inputFieldClassModifier} ${FormClassManager.getModifier(messageType)}`
-      : inputFieldClassModifier,
+    `af-form__${classNameSuffix}`,
+    modifiers,
+  );
+  const groupClassName = getComponentClassName(
+    className,
+    classModifier,
+    "af-form__group",
   );
 
   return (
     <div
-      className={classNames("row af-form__group", {
+      className={classNames("row", groupClassName, {
         "af-form__group--required": actualRequired,
+        "af-form__group--label-top": labelPosition === "top",
       })}
       role={roleContainer}
       aria-label={ariaLabelContainer}
+      aria-labelledby={isGroup ? labelId : undefined}
     >
       <div className={classNameContainerLabel}>
-        <label
+        <LabelElement
           className={classNames("af-form__group-label", {
             "af-form__group-label--required": actualRequired,
           })}
           htmlFor={isLabelContainerLinkedToInput ? inputId : undefined}
+          id={labelId}
         >
           {label}
-        </label>
+        </LabelElement>
       </div>
+
       <div className={classNameContainerInput}>
         <div className={fieldContainerClassName}>
-          {renderInput({
-            id: inputId,
-            classModifier: inputClassModifier,
-          })}
+          <Input
+            id={inputId}
+            classModifier={`${inputClassModifier} ${modifiers}`}
+            errorId={errorId}
+            ariaInvalid={isInvalid}
+            disabled={disabled}
+            {...otherProps}
+          />
```

Comment:
Ici le renderInput n'est pas considéré comme un composant react au sens strict il faut donc appel la fonction car cela pose des problèmes;, voir la PR #974
```suggestion
          {renderInput({
            id: inputId,
            classModifier: `${inputClassModifier} ${modifiers}`,
            errorId: errorId,
            ariaInvalid: isInvalid,
            disabled: disabled,
            ...otherProps
          })}
```

d'ailleurs à moment la doc react dit qu'il faut appel le renderInput en tant qu'element JSX
https://react.dev/reference/react/cloneElement#alternatives